ds701_book/07-Clustering-II-in-practice.qmd

```diff
@@ -18,11 +18,11 @@ __`scikit-learn.`__
 Our goals are to learn:
 
 ::: {.incremental}
-* How clustering is used in practice
-* Tools for evaluating the quality of a clustering
-* Tools for assigning meaning or labels to a cluster
-* Important visualizations
-* A little bit about feature extraction for text
+* How clustering is used in practice.
```

Comment:
I looked this up. The punctuation rule is that you only need it if it is a complete sentence. Not used for sentence fragments.  My preference is to leave them off fragments.

https://www.grammarly.com/blog/bullet-points/#:~:text=If%20the%20text%20of%20your,need%20to%20end%20with%20punctuation.